docs/decisions/0005-monitoring-by-code-owner.rst

```diff
@@ -0,0 +1,50 @@
+Monitoring by Owner
```

Comment:
Can we move this ADR to the new monitoring Django app?  As with other features in edx-platform, co-locating its documents/files with the feature's app is the direction we are headed.

Reply:
Done.